packages/tgc/src/common.components/search-input/search-input.tsx

```diff
@@ -0,0 +1,137 @@
+import { useLazyGetDisplayNameSuggestionsQuery } from 'platform-apis';
+import { IconSearch, Input, LINKS, NOTIFICATIONS_DURATION, SNACKBAR_TYPE } from 'platform-components';
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'wouter';
+
+import { clearChannelsState } from '../../store/slices/channels';
+import { clearMessages } from '../../store/slices/messages';
+import { getUserTypeSetting } from '../../store/slices/settings';
+import {
+    clearSuggestions, clearTwitchUser,
+    getIsSuggestionsLoading,
+    getIsTwitchUserFetching,
+    getSuggestions,
+} from '../../store/slices/twitch-user';
+import { getIsAuth } from '../../store/slices/user';
+import { addNotification, isValidSearchChange, isValidSearchSubmit } from '../../utils';
+
+import SearchInputSuggestions from './__suggestions/search-input__suggestions';
+
+const SearchInput = () => {
```

Comment:
cant find props for that component. Since that component is common, we need to get displayName from props, but it might be optional, so
```
interface SearchInputProps {
    displayName?: string;
} 
```
`const SearchInput = ({ displayName: propsDisplayName = '' }: SearchInputProps) => {`